### ISSUE TITLE
Create new blank wrapper failed with WS11WineCX64Bit20.0.4 + Wineskin-2.9.0.9

### ISSUE THREAD

**Issue by arzyu** [Gcenx/WineskinServer#155]
**Environment:**
 - OS: macOS 11.5.2
 - Winery version: 1.8.4.2
 - Master Wrapper version: Wineskin-2.9.0.9
 - Engine version: WS11WineCX64Bit20.0.4
 - Hardware: MacBook Air (13", Mid-2013)
 - Application/Game Name: 

**Describe the bug**
Create new blank wrapper failed with `WS11WineCX64Bit20.0.4` + `Wineskin-2.9.0.9`.
An extra file named `WS11WineCX64bit20.0.4.tar` was created in `~/Library/Application Support/Wineskin/Engines`

Error message:
```
The engine WS11WineCX64Bit20.0.4 is corrupted or opened incorrectly. If this error continues next time you try, reinstall the selected engine
```

Reinstalling the engine still doesn’t work

**Screenshots**
![Create new blank wrapper failed](https://user-images.githubusercontent.com/1270145/132459108-7e4f71ea-95bc-4118-ab76-cdf38e876fe9.png)

**Comment by Gcenx** [comment]
First update the master wrapper again to get the new versions.

Now goto `~/Library/Application Support/Wineskin/Engines` and trash anything things that's not a .tar.z7 file
Lastly goto /tmp and remove any thing that looks similar to `xWSxUsersxWSxgcenxxWSxApplicationsxWSxWineskinxWSxMyCoolWrapper`

Now with that cleaned up and with the new master wrapper 2.9.0.9-1 wrapper creation should be working again, I tested this before uploading this issue is related to your other now closed bug #154

**Comment by Gcenx** [comment]
Resolved via https://github.com/Gcenx/WineskinServer/commit/421ae0fad8a40f8ede351ed504767a70081b33e7

Just need to do the cleanup mentioned in the comment above this.
